### ISSUE TITLE
Quick purchase button or menu at planets and bases

### ISSUE THREAD

**Issue by valitas** [BartKieviet/Bookkeeper#21]
From Sweetness:
> any chance we can get a menu off to the left of a planet or star base that will allow us to buy avaialble upkeep for a building we track?

**Comment by BartKieviet** [comment]
I don't think this makes much sense, unless your food/water totals are below your cargo space which is almost never with the economies we're running ;)

**Comment by valitas** [comment]
I don't think I find it all that useful either.. but two users have asked for it now, and I think it's doable. Probably depends on #46 though so I'll look here again after that.

**Comment by valitas** [comment]
V2.2 sort of addresses this, in that you can get a quick peek at the building list right from the planet trade screen, and from that decide what you need to purchase. I'm not sure what else we could do about this, because I just can't imagine a simple button deciding what exactly I need. I mean, you don't usually buy for one building, you buy what you can carry to feed several buildings at once.

So, I'm going to close this really. If we can come up with a more specific idea of what to do, we can open a new ticket.
